### DISCUSSION TITLE
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods.

### DISCUSSION THREAD

**Discussion by charmjeh** [nuxt/nuxt#9630] [Questions (Nuxt 2)]
Hello, I'm new to nuxt.js and i was trying to setup my first nuxt.js project with 'yarn create nuxt-app'

and installed these, tried to use alias(@styles/*), etc.
- nuxt-property-decorator
- path
- vuetify-tsx

and i got this error.
 **"Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-private-methods."**

how do i fix this? or how do i find a clue?
now i deleted some project and tried to create new project with 'yarn create nuxt-app <project-name>' but it still doesn't work..

* node version : v14.15.4

**Comment by kissu** [comment]
Hi, what your Nuxt version?
Does this one help? https://stackoverflow.com/a/67353318/8816585

This is probably related to this btw: https://github.com/nuxt/nuxt.js/issues/9224#issuecomment-893224842

**Reply by charmjeh** [reply]
Hello kissu, thank you for your help :)
my nuxt.js version was 2.15.7.

i tried to set plugin and build option,
removed node modules, lock, .nuxt file and reinstalled,
but it didn't work

2.15.2 version works for me! 😄

**Reply by kissu** [reply]
Yep, creating a new project with `npx create-nuxt-app my-cool-project` does indeed have this warning.
So, let's say that this works if `2.15.5` < Nuxt version < `2.15.7`.

You can subscribe to the linked comment and track any info there for the latest update on the issue IMO. 😄 

Btw, this a **warning** and not an error, hence not critical if you need to do things, you can keep doing them.

**Reply by kissu** [reply]
Also, having this in your `nuxt.config.js` file is enough to remove the warning
```js
build: {
  babel: {
    plugins: [
      '@babel/plugin-proposal-class-properties',
      '@babel/plugin-proposal-private-methods',
    ],
  },
}
```

As shown [here](https://github.com/nuxt/nuxt.js/issues/9224#issuecomment-893224842)

**Reply by charmjeh** [reply]
Awesome, It works perfectly! Thanks a lot! 😆
